### DISCUSSION TITLE
Use Command in threads

### DISCUSSION THREAD

**Discussion by nguyenhuyentrang26602** [SudhanPlayz/Discord-MusicBot#601] [Ideas]
How can use command in threads?

**Comment by JotaroKujo0525** [comment]
> How can use command in threads?

Due to the nature of Discord API, Discord.js V12 don't allow that. so you’re gonna wanna either update the bot to discord.js13 _(Not recommended)_ or wait for v5 which will be in discord.js13

**Reply by nguyenhuyentrang26602** [reply]
When is v5 availabe?

**Reply by JotaroKujo0525** [reply]
> When is v5 availabe?

Nobody knows
